user_saml_shibboleth-php7.3/Dockerfile

```
@@ -15,8 +15,19 @@ RUN sed -i 's/updateSelinuxPolicy($inf);//g' /usr/lib64/dirsrv/perl/*
 ADD ldap/DSCreate.pm /usr/lib64/dirsrv/perl/DSCreate.pm
 ADD ldap/AdminServer.pm /usr/lib64/dirsrv/perl/AdminServer.pm
 RUN setup-ds-admin.pl --silent --file /ds-setup.inf
-RUN /usr/sbin/ns-slapd -D /etc/dirsrv/slapd-dir && sleep 20 && ldapmodify -H ldap:/// -f nextcloud.ldif -x -D "cn=Directory Manager" -w password && ldapadd -H ldap:/// -f users.ldif -x -D "cn=Directory Manager" -w password
+RUN /usr/sbin/ns-slapd -D /etc/dirsrv/slapd-dir \
+	# ensure to have enough time for startup
+	&& sleep 40 \
+	&& ldapmodify -H ldap:/// -f nextcloud.ldif -x -D "cn=Directory Manager" -w password \
+	&& ldapadd -H ldap:/// -f users.ldif -x -D "cn=Directory Manager" -w password \
+	&& sleep 10 \
+	&& kill -INT "$(ps -C ns-slapd -o pid h)" \
```

So, it looks like slapd is creating a file here with the process number: `/var/lock/dirsrv/slapd-dir/server`

Killing it with `INT` do not allow it to delete this file.
Killing it with `TERM` does.

```suggestion
 	&& kill -TERM "$(ps -C ns-slapd -o pid h)" \
```

Only tested it once, and the lock file is not here anymore so CI should be happier, crossing finger hard that it is 'simply' it :)